tutorials/plot_visualize_epochs.py

```diff
@@ -12,16 +12,20 @@
 data_path = op.join(mne.datasets.sample.data_path(), 'MEG', 'sample')
 raw = mne.io.read_raw_fif(op.join(data_path, 'sample_audvis_raw.fif'))
 raw.set_eeg_reference()  # set EEG average reference
-events = mne.read_events(op.join(data_path, 'sample_audvis_raw-eve.fif'))
-picks = mne.pick_types(raw.info, meg='grad')
-epochs = mne.Epochs(raw, events, [1, 2], picks=picks)
+events = mne.read_events(op.join(data_path, 'sample_audvis_raw-eve.fif'),
+                         include=[5, 32])
+epochs = mne.Epochs(raw, events, 5, tmin=-0.2, tmax=1.)
```

Comment:
using 5 makes all the other plots not nice.

you should show the other plots using event_id 1 or 2 like before